site/content/docs/5.2/components/dropdowns.md

```diff
@@ -696,6 +716,13 @@ Separate groups of related menu items with a divider.
 
```

Comment:
❓ Ask design if dropend variant will be added in ODS and if ok to let without callout msg.

**TO DO** : Look for "Is the phrase "Feel free to style further with custom CSS or text utilities." really true ? ;-) We are just adding warnings to avoid developers to use unauthorized variants..." sentence to delete it because no compatible with the callout messages. (message that comes from Bootstrap)